mdpow/workflows/registry.py

```diff
@@ -0,0 +1,31 @@
+# MDPOW: registry.py
+# 2023 Cade Duckworth
+
+"""
+:mod:`mdpow.workflows.registry` --- Registry of currently supported automated workflows
+=======================================================================================
+
+Each entry in :mod:`mdpow.workflows.registry` corresponds to an :class:`~mdpow.analysis.ensemble.EnsembleAnalysis`
+for which exists a corresponding automated workflow.
+
+Currently supported workflows - :mod:`~mdpow.workflows.dihedrals`
```

Comment:
Does not format as a list, see https://mdpow--229.org.readthedocs.build/en/229/workflows/registry.html#mdpow-workflows-registry-registry-of-currently-supported-automated-workflows
![image](https://user-images.githubusercontent.com/237980/228390948-05ac5cf2-a3ac-4e0e-8d80-ad32087e9bae.png)

Can you please create a [reST table](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#tables) (and see more under [table role](https://docutils.sourceforge.io/docs/ref/rst/directives.html#table)) that looks like

| **keyword** | **analysis function** |
|---------|---------|
| DihedralAnalysis | :func:'dihedrals.automated_dihedral_analysis <mdpow.workflows.dihedrals.automated_dihedral_analysis>' |

where the keyword is the string needed for :func:'mdpow.workflows.base.automated_project_analysis' and the analysis function is the actual automated function that will be executed — essentially the same information that is in the registry dict but immediately understandable (and usable) by humans.

For the reST, should look something like (include [name](https://docutils.sourceforge.io/docs/ref/rst/directives.html#common-options) for hyperlinking)
```reST
.. table:: Available automated workflows.
   :widths: auto
   :name: workflows_table

    ==============  ============
    keyword         function
    ==============  ============
    ...
    ...
    ==============  ============
```